CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 ## Unreleased
 
+### Recommended changes
```

Comment:
@calvin-lau-sig7 do you want to take a look at these release notes and let me know if you have any comments/suggestions?